--- gpgme-0.4.3/configure.ac.orig	Mon Oct  6 20:30:54 2003
+++ gpgme-0.4.3/configure.ac	Mon Dec  8 12:18:36 2003
@@ -247,7 +247,7 @@
   AC_CHECK_FUNCS(funopen)
   if test $ac_cv_func_funopen != yes; then
     # No funopen but we can implement that in terms of fopencookie.
-    AC_CHECK_FUNCS(fopencookie, AC_LIBOBJ(funopen),
+    AC_CHECK_FUNCS(fopencookie, [AC_LIBOBJ(funopen)],
                    AC_MSG_ERROR([[
 No implementation of fopencookie or funopen available.
 ]]))
