--- gpgme-1.8.0/lang/python/Makefile.am.orig	2016-11-27 12:56:09.390089844 +0100
+++ gpgme-1.8.0/lang/python/Makefile.am	2016-11-27 17:00:38.506589016 +0100
@@ -101,14 +101,15 @@
 	rm -f install_files.txt
 	for PYTHON in $(PYTHONS); do \
 	  $$PYTHON setup.py install \
-	  --prefix $(DESTDIR)$(prefix) \
+	  --prefix $(prefix) \
+	  --root $(DESTDIR) \
+	  --optimize=2 \
 	  --record files.txt \
 	  --verbose ; \
 	  cat files.txt >> install_files.txt ; \
 	  rm files.txt ; \
 	done
 	$(MKDIR_P) $(DESTDIR)$(pythondir)/gpg
-	mv install_files.txt $(DESTDIR)$(pythondir)/gpg
 
 uninstall-local:
 	xargs <$(DESTDIR)$(pythondir)/gpg/install_files.txt -- rm -rf --
