--- gpgme-1.7.0/lang/python/Makefile.am.orig	2016-09-21 09:18:00.000000000 +0200
+++ gpgme-1.7.0/lang/python/Makefile.am	2016-10-08 22:07:19.258156918 +0200
@@ -92,14 +92,15 @@
 	rm -f install_files.txt
 	for PYTHON in $(PYTHONS); do \
 	  $$PYTHON setup.py install \
-	  --prefix $(DESTDIR)$(prefix) \
+	  --prefix $(prefix) \
+	  --root $(DESTDIR) \
+	  --optimize=2 \
 	  --record files.txt \
 	  --verbose ; \
 	  cat files.txt >> install_files.txt ; \
 	  rm files.txt ; \
 	done
 	$(MKDIR_P) $(DESTDIR)$(pythondir)/pyme
-	mv install_files.txt $(DESTDIR)$(pythondir)/pyme
 
 uninstall-local:
 	xargs <$(DESTDIR)$(pythondir)/pyme/install_files.txt -- rm -rf --
