--- gpgme-1.9.0/lang/qt/src/dn.cpp.orig	2017-03-20 20:10:15.000000000 +0100
+++ gpgme-1.9.0/lang/qt/src/dn.cpp	2017-03-29 20:58:29.893277570 +0200
@@ -167,7 +167,7 @@
     for (unsigned int i = 0; i < numOidMaps; ++i)
         if (!strcasecmp((char *)p, oidmap[i].oid)) {
             free(p);
-            gpgrt_asprintf(&p, oidmap[i].name);
+            gpgrt_asprintf(&p, "%s", oidmap[i].name);
             break;
         }
     array->key = p;
