--- gpgme-1.0.0/configure.ac.orig	2004-09-30 03:41:06.000000000 +0200
+++ gpgme-1.0.0/configure.ac	2004-10-17 22:04:07.863356864 +0200
@@ -330,7 +330,7 @@
   AC_DEFINE_UNQUOTED(GPGSM_PATH, "$GPGSM", [Path to the GPGSM binary.])
   AC_SUBST(GPGSM)
 fi
-AM_CONDITIONAL(HAVE_GPGSM, [test -n "$GPGSM"])
+AM_CONDITIONAL(HAVE_GPGSM, [test -n "$GPGSM" -a "$GPGSM" != "no"])
 dnl Check for GPGSM version requirement.
 GPGSM_VERSION=unknown
 ok=maybe
