--- gpgme-0.4.3/gpgme/gpgme.m4.orig	2003-09-03 03:15:56.000000000 +0200
+++ gpgme-0.4.3/gpgme/gpgme.m4	2003-12-30 04:39:40.899689648 +0100
@@ -1,7 +1,7 @@
 dnl Autoconf macros for libgpgme
 dnl $Id$
 
-AC_DEFUN(_AM_PATH_GPGME_CONFIG,
+AC_DEFUN([_AM_PATH_GPGME_CONFIG],
 [ AC_ARG_WITH(gpgme-prefix,
             AC_HELP_STRING([--with-gpgme-prefix=PFX],
                            [prefix where GPGME is installed (optional)]),
